composer.json

```diff
@@ -74,6 +75,12 @@
   "scripts": {
     "build": "@phar:build",
     "test": "@test:functional",
+    "phar:install-tools": [
+      "gem install mime-types -v 2.6.2",
+      "curl -LSs https://box-project.github.io/box2/installer.php | php",
+      "mkdir -p tools",
+      "mv -f box.phar tools/box"
+    ],
```

Comment:
I put this stuff here while trying to get the unit tests to run.. but I don't know that it's actually needed for them